### DISCUSSION TITLE
How to hide other users' email addresses?

### DISCUSSION THREAD

**Discussion by leonardodipace** [directus/directus#13036] [Feature Requests]
Is there any way to hide the email addresses of other registered users when fetching the API?
In my case the currently logged in user should only be able to see the first name and the last name of the other users, but still being able to see all of his/her personal information.
I hope someone can help me!

**Comment by rashidpathiyil** [comment]
I think currently not possible setup field permissions for system fields.

![Screenshot_2022-04-30-02-07-03-825_com.brave.browser.jpg](https://user-images.githubusercontent.com/15140041/166066434-37fddcee-6e61-42be-808a-76487aafdd7e.jpg)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
